assignments/pig-latin/code.js

```diff
@@ -1,33 +1,9 @@
-// A new method you will probably want to use.
-//
-// If s is a string:
-//
-//  s.search(/[aeiou]/) returns the index in s of the first vowel.
-//
-// For instance
-//
-//  const s = 'phone';
-//
-//  s.search(/[aeiou]/) returns 2
-//
-// The funny syntax /[aeiou]/ is called a "regular expression". We'll learn
-// about them in more detail later.
-//
-// To write the advancedPigLatin function you will want to know about a new
-// operator called the "ternary operator". (It's called this because it has
-// three operands.)
-//
-// It looks like:
-//
-//  booleanValue ? value1 : value2
-//
-// And it evaluates to value1 is booleanValue is true and value2 if it is false.
-//
-// For example:
-//
-//  x === 0 ? "Zero" : "Not zero"
-//
-// will evaluate to either "Zero" or "Not zero" depending on whether x === 0.
-//
-// Feel free to play around in the REPL to make sure you understand how this
-// operator works.
+const pigLatin = (s) => {
+  let v = s.search(/[aeiou]/)
```

Comment:
Anywhere you define a variable with `let` that never changes (i.e. is never assigned to again) you can define it instead with `const`.

```suggestion
  const v = s.search(/[aeiou]/)
```